fairing/constants/constants.py

```diff
@@ -8,9 +8,10 @@
 DEFAULT_GENERATED_DOCKERFILE_FILENAME = '/tmp/Dockerfile'
 
 GOOGLE_CREDS_ENV = 'GOOGLE_APPLICATION_CREDENTIALS'
-GCP_CREDS_SECRET_NAME = 'user-gcp-sa'
 
+GCP_CREDS_SECRET_NAME = 'user-gcp-sa'
 AWS_CREDS_SECRET_NAME = 'aws-secret'
+DOCKER_CREDS_SECRET_NAME = "docker-secret"
```

Comment:
Why not let user inputs the secret name?

Reply:
Yes that could be done. Currently gcp secret name is also hard code. I think a task could be created for inputting all the secret names and cod enhanced in one go with agreed strategy.

Reply:
ye, shall we just let user point out the secret name? and fairing will mount the secret with application. seems that's more general.

Reply:
will be ok to to auto create secret for now.